Backend/router/v1/report.ctrl.js

```diff
@@ -10,32 +10,42 @@ const createReport = (req, res) => {
   }).then((result) => {
     res.status(201);
     res.send("good");
+  }).catch((err) => {
+    res.json({msg: "error"});
```

Comment:
Can you be more specific in error string?
"error" doesn't help to let other know what is happening.